Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
+
+all: image test
```

Comment:
This is a holdover from the one I wrote for builder, with the idea being that just typing `make` would build an image & also run tests to confirm goodness. You can, of course, run `make test` explicitly.

I'm fine with removing it & saying the default task should simply be building the image. I'll update builder as well if that's preferred, since whichever way we do it should be consistent.

Actually, I'll go ahead and remove it in both places: earlier Makefiles (app, toolbox) don't do this.